packages/graph/src/graph2/graph.ts

```diff
@@ -209,6 +200,22 @@ export class Graph2 extends SVGZoomWidget {
         this.zoomToFitLimit(1);
     }
 
+    resolveAnnoEventNode(): undefined | IVertex4Annotation {
+        const d3evt = d3Event();
+        const eventPath = d3evt?.sourceEvent?.path ?? d3evt.path;
+        const anno = eventPath.find(n => {
+            if(typeof n.hasAttribute === "function") {
```

Comment:
Is this test needed?

Reply:
It may not be needed, but it's safer than trusting that other browsers won't insert weird things into that path array.